src/rgw/rgw_main.cc

```diff
@@ -571,6 +572,7 @@ int main(int argc, const char **argv)
 
   rgw_tools_cleanup();
   rgw_shutdown_resolver();
+  rgw_release_all_curl_handles();
```

Comment:
in future, for master & L we could move this under rgw::curl namespace introduced in https://github.com/ceph/ceph/pull/20390